muikku-core-plugins/src/main/resources/META-INF/resources/dust/announcer/announcer_item.dust

```diff
@@ -1,5 +1,5 @@
 
-<div class="mf-item an-announcement">
+<div class="mf-item an-announcement" id="{id}">
```

Comment:
it's bad practice to use database row id as html element id, maybe you should use data-announcement-id instead?